static/templates/sidebar/item-document-partial.hbs

```diff
@@ -11,7 +11,7 @@
                 <span class="item-level">{{localize "PF2E.Item.Spell.LevelN" level=this.level}}</span>
             {{else if (eq this.type "effect")}}
                 <span class="item-level">{{localize "PF2E.LevelN" level=this.level}}</span>
-            {{else}}
+            {{else if this.level includeZero=true}}
```

Comment:
obscene

Reply:
apparently null is greater than or equal to 0 :(

Reply:
there's an `isNumber` helper

Reply:
I should probably replace the main if with this, since greater than or equal to zero matching null is problematic in the first place, and negative levels aren't a thing we do to remove them.

Reply:
includeZero is part of handlebars, and is used when you want an if statement to check that something has a value rather than just if its falsey. Regardless, main if.